content/general/presence.textile

```diff
@@ -0,0 +1,13 @@
+---
+title: Presence
+section: general
+index: 27
+---
+
+Ably's presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel) in real time using our "realtime SDKs":/realtime, or via the "Reactor service":https://www.ably.io/reactor. You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the "REST API":/rest-api#presence.
+
+<a href="/images/diagrams/presence.png" target="_blank">
+  <img src="/images/diagrams/presence.png" style="width: 100%" alt="Presence representation">
```

Comment:
For the image, by itself it doesn't really explain anything, looking at it without knowing what it's meant to mean I can't tell. A caption at the least would be good to give some context to the image.

If we're trying to indicate that you can see who is offline/online, perhaps an image helping to demonstrate the fact there's communication between Ably and connected devices?

An extremely rough sketch would be:

<img width="407" alt="presencelol" src="https://user-images.githubusercontent.com/9784119/43513014-93e9a160-9574-11e8-8e08-45636b06ef5b.png">

Not really sure the best way to indicate it, if you can think of anything @Srushtika that'd be great 😄

Reply:
That's a great suggestion, will try something very similar to your sketch! :)